charts/datadog/values.yaml

```diff
@@ -374,6 +374,10 @@ datadog:
     # datadog.networkMonitoring.enabled -- Enable network performance monitoring
     enabled: false
 
+  serviceMonitoring:
+    # datadog.serviceMonitoring.enabled -- Enable Universal Service Monitoring
+    enabled: false
+
```

Comment:
Should it be under `datadog.networkMonitoring` ? is it consider as an extension of NPM? 

my idea was to have something like.

```yaml
  ## Enable network monitoring and provide granular configs
  networkMonitoring:
    # datadog.networkMonitoring.enabled -- Enable network performance monitoring
    enabled: false
    # Enable network monitoring and provide granular configuration
    serviceMonitoring:
      # datadog.networkMonitoring.serviceMonitoring.enabled -- Enable Universal Service Monitoring
      enabled: false
      # TODO: add configuration option related to serviceMonitoring
```

Reply:
It is a separate product from NPM and billed separately.